Scalar/CommandLine/UpgradeVerb.cs

```diff
@@ -142,13 +150,53 @@ private bool TryInitializeUpgrader(out string error)
             }
         }
 
+        private bool TryUpgradeWithBrew(out string error)
+        {
+            this.Output.WriteLine("Running 'brew update'");
+
+            var launcher = new ProcessLauncher();
+
+            bool result = launcher.TryStart("brew", "update", useShellExecute: false, out Exception ex);
+
+            if (!result) {
+                this.tracer.RelatedEvent(EventLevel.Warning, "Failure during 'brew update'", this.CreateEventMetadata(ex));
+            }
+
+            launcher.WaitForExit();
+
+            string packageName = "scalar";
```

Comment:
Should these Cask names be pulled out to a constants class (somewhere)?

Reply:
Is there any chance we will use these constants a second time? Putting them in an `enum` local to this class might make sense, especially with your recommendation to scan the `brew cask outdated` / `brew cask list` output as well. That will make multiple uses _in this file_ but nowhere else. Thanks.